src/test/java/com/qunite/api/service/QueueServiceTest.java

```diff
@@ -270,4 +271,32 @@ void testDeleteMemberFromQueueConcurrent() throws InterruptedException {
       fail("unexpected actualEntryIdList size");
     }
   }
+
+  @Sql({"/users-create.sql", "/queues-create.sql"})
+  @Test
+  void testAddingManager() {
+    var queueId = 1L;
+    var managerId = 3L;
+    var queueCreatorUsername = "First";
+
+    queueService.addManager(managerId, queueId, queueCreatorUsername);
+    var managersList = userRepository.findAllByManagedQueuesId(queueId);
+    var manager = JpaRepositoryUtils.getById(managerId, userRepository);
+
+    assertThat(managersList).containsOnly(manager);
+  }
+
+  @Sql({"/users-create.sql", "/queues-create.sql", "/queues-managers-create.sql"})
+  @Test
+  void testDeletingManager() {
+    var queueId = 1L;
+    var managerId = 3L;
+    var queueCreatorUsername = "First";
+
+    queueService.deleteManager(managerId, queueId, queueCreatorUsername);
+    var managersList = userRepository.findAllByManagedQueuesId(queueId);
+    var manager = JpaRepositoryUtils.getById(managerId, userRepository);
```

Comment:
Why to introduce separate REPOSITORY method for this and other tests?
You can just use:
```
var queue = JpaRepositoryUtils.getById(1L, queueRepository);
assertThat(queue.getManagers()).doesNotContain(manager);
```